ErinUntermeyer/Refactor-Tractor-FitLit-B
# Integrate post

###### What's this PR do?
- Updated the index.html file with a form for posting data for Hydration, Sleep and Activity
- Wrote function to format the data taken from the form input
- Wrote the function to POST the data and called the format data functions
- Added an event listener to call the post data function when the Post it! button is clicked
- Removes data files that are not used anymore
###### Where should the reviewer start?
open index.html to see the new forms!
###### How should this be manually tested?
open index.html and submit some data. You can then visit the APIs to see the new data posted.
###### Any background context you want to provide?
We worked on all of these features together, everyone is aware of all changes
###### What are the relevant tickets?
Closes #65 , closes #46 , closes #41
